src/index.jsx

```diff
@@ -34,7 +34,9 @@ const store = getStore();
 
 root.render(
   <Provider store={store}>
-    <BrowserRouter>
+    <BrowserRouter
+      future={{ v7_relativeSplatPath: true, v7_startTransition: true }}
```

Comment:
These flags are not needed on v7:

> breaking changes are introduce din minor releases behind a flag, allowing users to opt-in at their convenience. In the next major release, all future flag behaviors become the default behavior.

Source: https://github.com/remix-run/react-router/blob/main/CHANGELOG.md#adopted-future-flag-behaviors